### DISCUSSION TITLE
Please tell me about your favorite artists! (any genre) :o

### DISCUSSION THREAD

**Discussion by hisashi-y** [NAIST-SE/AttractingContributors#47] [Q&A]
Come to my fork:
https://github.com/hisashi-y/AttractingContributors-FavoriteArtists

Add your favorite artists to the README file! Any genre is appreciated.

**Comment by mailtoy** [comment]
> Come to my fork:
> https://github.com/hisashi-y/AttractingContributors-FavoriteArtists
> 
> Add your favorite artists to the README file! Any genre is appreciated.

I added my favorite artists to your readme :)

**Comment by tattiyaS** [comment]
Hope you like the artist i added:D

**Comment by noppadol-assava** [comment]
I commit pull request on your repo. 
Please help me adding Food into the list.
https://github.com/NoppadolAssava/AttractingContributors---Food

**Comment by vittunyuta-aom** [comment]
added!! please help me https://github.com/aommoaGitHub/SuperFavoriteSongs

**Comment by haxhimitsu** [comment]
Thanks merge:)
